app/containers/AskQuestion/AskQuestionForm.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { FormattedMessage } from 'react-intl';
+
+import LoadingIndicator from 'components/LoadingIndicator';
+
+import messages from './messages';
+import { strLength20, required } from './validate';
+import { FORM_TITLE, FORM_CONTENT } from './constants';
+
+import renderTextInput from './renderTextInput';
+import renderTextarea from './renderTextarea';
+
+import Box from './Box';
+
+const AskQuestionForm = props => (
+  <Box onSubmit={props.handleSubmit(props.postQuestion)}>
+    <h4 className="header">
+      {props.translations[messages.title.id].toUpperCase()}
```

Comment:
No need to convert to upper case in JavaScript. It is styling and should be done in CSS.